ksql-common/src/test/java/io/confluent/ksql/util/SchemaUtilTest.java

```diff
@@ -295,44 +295,4 @@ public void shouldReturnFieldNameWithoutAliasAsIs() {
     assertThat("Invalid field name", SchemaUtil.getFieldNameWithNoAlias(schema.fields().get(0)),
         equalTo(schema.fields().get(0).name()));
   }
-
-  @Test
-  public void shouldCreateCorrectSchemaDescription() {
-    Schema addressSchema = SchemaBuilder.struct()
-        .field("NUMBER", Schema.INT64_SCHEMA)
-        .field("STREET", Schema.STRING_SCHEMA)
-        .field("CITY", Schema.STRING_SCHEMA)
-        .field("STATE", Schema.STRING_SCHEMA)
-        .field("ZIPCODE", Schema.INT64_SCHEMA)
-        .build();
-
-    SchemaBuilder schemaBuilder = SchemaBuilder.struct();
-    Schema structSchema = schemaBuilder
-        .field("ordertime", Schema.INT64_SCHEMA)
-        .field("orderid", Schema.INT64_SCHEMA)
-        .field("itemid", Schema.STRING_SCHEMA)
-        .field("orderunits", Schema.FLOAT64_SCHEMA)
-        .field("arraycol",schemaBuilder.array(Schema.FLOAT64_SCHEMA))
-        .field("mapcol", schemaBuilder.map(Schema.STRING_SCHEMA, Schema.FLOAT64_SCHEMA))
-        .field("address", addressSchema).build();
-
-    String schemaDescription = SchemaUtil.describeSchema(structSchema);
-
-    assertThat(schemaDescription, equalTo("STRUCT < \n"
```

Comment:
is there a reason we removed this test? how do we test that the cli output is what we expect it to be?

Reply:
The function under test was removed. The cli output is tested by the cli test. Now that you mention it though that test doesn't have a struct. So I'll add that.